__tests__/__snapshots__/renderSpec.jsx.snap

```diff
@@ -0,0 +1,257 @@
+              />
+            </g>
+          </svg>
+          <a
+            className="directions cardText"
+            href="https://maps.google.com/?q=51052 Roob Forge West Chesley Utah, 11729-3188"
+          >
+            Get Directions
+          </a>
+           
+        </li>
+      </ul>
+    </div>
+  </div>
+</div>
+`;
+
+exports[`Stars Component render it renders correctly 1`] = `
+<div>
+  <div
+    className="starContainer"
+  >
+    <div
+      className="redStar star"
+    >
+      <img
+        alt="star img"
+        className="starImage"
+        src="test-file-stub"
+      />
+    </div>
+    <div
+      className="redStar star"
+    >
+      <img
+        alt="star img"
+        className="starImage"
+        src="test-file-stub"
+      />
+    </div>
+    <div
+      className="redStar star"
+    >
+      <img
+        alt="star img"
+        className="starImage"
+        src="test-file-stub"
+      />
+    </div>
+    <div
+      className="redStar star"
+    >
+      <img
+        alt="star img"
+        className="starImage"
+        src="test-file-stub"
+      />
+    </div>
+    <div
+      className="greyStar star"
+    >
+      <img
+        alt="star img"
+        className="starImage"
+        src="test-file-stub"
+      />
+    </div>
+    <div
+      className="reviewText"
+    >
+      3345 reviews
+    </div>
+  </div>
+</div>
+`;
+
+exports[`Title Component render it renders correctly 1`] = `
+<div>
+  <div
+    className="title"
+  >
+    Nienow, Bradtke and Hills
+  </div>
+  <div>
+    <div
+      className="starContainer"
+    >
+      <div
+        className="redStar star"
+      >
+        <img
+          alt="star img"
+          className="starImage"
+          src="test-file-stub"
+        />
+      </div>
+      <div
+        className="redStar star"
+      >
+        <img
+          alt="star img"
+          className="starImage"
+          src="test-file-stub"
+        />
+      </div>
+      <div
+        className="redStar star"
+      >
+        <img
+          alt="star img"
+          className="starImage"
+          src="test-file-stub"
+        />
+      </div>
+      <div
+        className="redStar star"
+      >
+        <img
+          alt="star img"
+          className="starImage"
+          src="test-file-stub"
+        />
+      </div>
+      <div
+        className="greyStar star"
+      >
+        <img
+          alt="star img"
+          className="starImage"
+          src="test-file-stub"
+        />
+      </div>
+      <div
+        className="reviewText"
+      >
+        3345 reviews
+      </div>
+    </div>
+  </div>
+</div>
```

Comment:
This is a huge file I am a little unfamiliar with so I would suggest maybe have one other person look at it who might be able to see things I missed.

Reply:
It's basically a snapshot file that jest makes to use in it's testing and to compare rendered data to while using the snapshot testing methods. It advises to commit and include in code review on the docs so i'm including it here.. Made my PR huge though.